langserver/references.go

```diff
@@ -142,7 +157,10 @@ func (h *LangHandler) handleTextDocumentReferences(ctx context.Context, conn JSO
 		log.Printf("info: timeout during references for %s, found %d refs", defpkg, len(locs))
 	}
 
-	if findRefErr != nil {
+	// If we find references then we can ignore findRefErr. It should only
+	// be non-nil due to timeouts or our last findReferences doesn't find
+	// the def.
+	if len(locs) == 0 && findRefErr != nil {
```

Comment:
Does this mean an LSP error occurs if we find zero references? That doesn't sound like an error to me.